CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased
 
+## [0.1.5] - 2022-09-30
+### Added
+- Add arm builds for images from [meain](https://github.com/meain)
```

Comment:
Is this change related to the helm-chart?

Reply:
Not really, but since this is a major change in deployment and since there is no other specific change, I thought this could go in here.

Reply:
Dropped the entry.